ldk/javascript/examples/self-test-loop/src/tests/screen/index.ts

```diff
@@ -109,3 +111,97 @@ export async function performOcr() {
       });
   });
 }
+async function testOCRUsingCursor() {
+  let i = 0;
+  let cursorPositionStream: Cancellable;
+  let topParam: number;
+  let leftParam: number;
+  let topParam1: number;
+  let leftParam1: number;
+  await sleep(1000);
+  await cursor.listenPosition((response) => {
+    if (typeof response !== 'undefined') {
+      if (i === 0) {
+        topParam = response.y;
+        leftParam = response.x;
+        console.debug(`leftParam - X - ${leftParam1}`);
+        console.debug(` topParam - Y - ${topParam1}`);
+      }
+      if (i === 1) {
+        topParam1 = response.y;
+        leftParam1 = response.x;
+        console.debug(`leftParam1 - X - ${leftParam1}`);
+        console.debug(` topParam1 - Y - ${topParam1}`);
+      }
+
+      i += 1;
```

Comment:
This is really my only issue with the PR, listenPosition has a .5 second polling time. I'd rather do something like have the user hit spacebar to capture the cursor position, then hit spacebar again to capture the second point instead of polling twice on the listener.

Reply:
@MichaelOliveMcCarrick  Sry for the late reply, yeah, I think that's a better way to initiate the process!